### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by toomey8**
Ready for review @thoughtbot/growth

### CODE REVIEW

File: README.md
Comment (croaky) [inline README.md:88]: The `utm_source` on thoughtbot.com is often captured in the session and if the user submits a form, we store the source in the CRM, so can run reports on the last touch all the way through the end of the sales pipeline for projects that close. Not a big deal to remove it if you'd like, but maybe we use the param consistently one way or the other on these links?

File: README.md
Comment (croaky) [inline README.md]: Is this link orphaned now? Can it be removed? Or, keep `[our other projects][community]` above and edit this line to be `https://thoughtbot.com/tools?utm_source=github` or `https://thoughtbot.com/tools`?

File: README.md
Comment (croaky) [inline README.md:85]: Not sure if the maintainers on this project care about a hard wrap at 80 characters but something to keep an eye on as you edit these, can usually tell by the context around it.

**Review by croaky — APPROVED** [review]
Looks good to me.

File: README.md
Comment (toomey8) [reply README.md:88]: Would it be ok to remove the params altogether? I don't mind leaving them if there is a reason, but for analytics it would be better to remove them all.

File: README.md
Comment (toomey8) [reply README.md]: Sure, I'll send it to /tools provisionally

File: README.md
Comment (gfontenot) [reply README.md:85]: > Not sure if the maintainers on this project care about a hard wrap at 80 characters

Yes, please wrap this to 80 characters and use the markdown link syntax we're already using elsewhere in the document.

Question: Is this something we actively want to promote? My understanding is that we aren't actively producing content for Upcase related to iOS. If that's the case, I'd rather not add a link to it here.

File: README.md
Comment (gfontenot) [inline README.md:87]: This link got broken

Comment (gfontenot) [reply README.md:87]: Actually this link isn't being used anymore? I believe this should be changed to be the `[tools]` link.

**Review by gfontenot — CHANGES_REQUESTED** [review]

**Comment by gfontenot** [comment]
I'd like to make sure these commits get squashed/rebased/reworded appropriately before merging this into master.

File: README.md
Comment (croaky) [reply README.md:85]: As an iOS developer, does the content on https://thoughtbot.com/upcase/ios still look valuable to you? If it's evergreen, it will be useful to someone who comes across it and linking to it from our iOS repos helps people find it.

If it's not valuable, let's work with the Upcase team to pull it down.

How actively we're producing new content on Upcase doesn't need to influence the decision.

File: README.md
Comment (croaky) [inline README.md:89]: It looks like the "Hire Us" link got removed? Think it would still be good for people checking out this part of the README to know we're available for hire for iOS development.

**Comment by gfontenot** [comment]
I'm going to go ahead and close this due to inactivity but please feel free to reopen if this is still an issue.

**Pull request closed without merging**
